addons/l10n_cl/models/account_move.py

```diff
@@ -160,6 +161,36 @@ def _l10n_cl_get_invoice_totals_for_report(self):
             tax_totals['formatted_amount_untaxed'] = formatLang(self.env, tax_totals['amount_untaxed'], currency_obj=self.currency_id)
             if tax_totals['subtotals']:
                 tax_totals['subtotals'][0]['formatted_amount'] = tax_totals['formatted_amount_untaxed']
+        values = self._l10n_cl_get_amounts()
+        withholdings = self._l10n_cl_get_withholdings()
+        if self.currency_id != self.env.ref('base.CLP'):
+            tax_totals['formatted_amount_total'] = formatLang(
+                self.env, values['total_amount'], currency_obj=self.env.ref('base.CLP'))
+            tax_totals['subtotals'][0]['amount'] = values['subtotal_amount_taxable']
+            tax_totals['subtotals'][0]['formatted_amount'] = formatLang(
+                self.env, values['subtotal_amount_taxable'], currency_obj=self.env.ref('base.CLP'))
+        list_of_tax_groups = []
+        for groups_by_subtotal in tax_totals['groups_by_subtotal']['Base imponible']:
+            if groups_by_subtotal['tax_group_name'] != 'IVA 19%':
+                continue
+            groups_by_subtotal['formatted_tax_group_amount'] = formatLang(
+                self.env, values['vat_amount'], currency_obj=self.env.ref('base.CLP'))
+            groups_by_subtotal['formatted_tax_group_base_amount'] = formatLang(
+                self.env, values['subtotal_amount_taxable'], currency_obj=self.env.ref('base.CLP'))
+            list_of_tax_groups.append(groups_by_subtotal)
+        for withholding in withholdings:
+            list_of_tax_groups.append({
+                'tax_group_name': withholding['tax_name'],
+                'formatted_tax_group_amount': formatLang(self.env, withholding['tax_amount'], currency_obj=self.env.ref('base.CLP')),
+                'formatted_tax_group_base_amount': formatLang(self.env, withholding['tax_base'], currency_obj=self.env.ref('base.CLP'))
+            })
+        tax_totals['groups_by_subtotal']['Base imponible'] = list_of_tax_groups
+        if values['subtotal_amount_exempt']:
+            tax_totals['subtotals'].append({
+                'amount': values['subtotal_amount_exempt'],
+                'formatted_amount': formatLang(self.env, values['subtotal_amount_exempt'], currency_obj=self.env.ref('base.CLP')),
+                'name': 'Monto Exento'})
```

Comment:
```suggestion
                'name': _("Exempt Amount")})
```

Reply:
I made this commit, but at least locally, it does not work for me.

Reply:
b7944a0c